src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -0,0 +1,137 @@
+        :subtitle max-fees
+        :loading? loading-fees?
+        :size     :small}]
+      [data-item
+       {:title         (i18n/label :t/max-slippage)
+        :subtitle      max-slippage
+        :subtitle-icon :i/edit
+        :loading?      loading-fees?}]]]))
+
+(defn view
+  []
+  (let [[pay-value set-pay-value]        (rn/use-state "")
+        {:keys [color]}                  (rf/sub [:wallet/current-viewing-account])
+        swap-transaction-data            (rf/sub [:wallet/swap])
+        currency                         (rf/sub [:profile/currency])
+        currency-symbol                  (rf/sub [:profile/currency-symbol])
+        asset-to-pay                     (rf/sub [:wallet/swap-asset-to-pay])
+        asset-to-receive                 (rf/sub [:wallet/swap-asset-to-receive])
+
+        {:keys [max-slippage swap-proposal loading-fees?
+                receive-amount network]} swap-transaction-data
+        pay-token-fiat-value             (utils/calculate-token-fiat-value
+                                          {:currency currency
+                                           :balance  (or pay-value 0)
+                                           :token    asset-to-pay})
+        receive-token-fiat-value         (utils/calculate-token-fiat-value
+                                          {:currency currency
+                                           :balance  (or receive-amount 0)
+                                           :token    asset-to-receive})
+        native-currency-symbol           (get-in swap-proposal
+                                                 [:from :native-currency-symbol])
+        pay-token-symbol                 (:symbol asset-to-pay)
+        receive-token-symbol             (:symbol asset-to-receive)
+        on-press                         (fn [v] (set-pay-value (str pay-value v)))
+        delete                           (fn []
+                                           (set-pay-value #(subs % 0 (dec (count %)))))]
+    [rn/view {:style style/container}
+     [account-switcher/view
+      {:on-press      navigation/navigate-back
+       :icon-name     :i/arrow-left
+       :margin-top    (safe-area/get-top)
+       :switcher-type :select-account}]
+     [rn/view {:style style/inputs-container}
+      [quo/swap-input
+       {:type                 :pay
+        :error?               false
+        :token                pay-token-symbol
+        :customization-color  :blue
+        :show-approval-label? false
+        :status               :default
+        :currency-symbol      currency-symbol
+        :on-swap-press        #(js/alert "Swap Pressed")
+        :on-token-press       #(js/alert "Token Pressed")
+        :on-max-press         #(js/alert "Max Pressed")
+        :value                pay-value
+        :fiat-value           receive-token-fiat-value
+        :network-tag-props    {:title    (str (i18n/label :t/max {:number 200}) " " pay-token-symbol)
```

Comment:
Can add the token symbol through interpolation, same as the amount.

Reply:
Sure, will do!